components/collections/collection.vue

```diff
@@ -8,7 +8,7 @@
       @drop="dragging = false"
       @dragleave="dragging = false"
       @dragend="dragging = false"
-      @click="$emit('select-folder', { name: '', id: collection.id })"
+      @click="$emit('select-folder', { name: '', id: collection.id, reqIdx: '' })"
     >
       <button class="icon" @click="toggleShowChildren">
```

Comment:
```
      <button class="icon" @click="toggleShowChildren">
```
Button component already has `@click="toggleShowChildren"` trigger. `select-folder` trigger should be appended to this `@click`.

Reply:
Also on a second thought, it's efficient to trigger the new `select-folder` trigger only if the component is inside save request modal. I mean there's a flag that tells `<Collection />` component that it's being displayed inside save request modal right? Use that to conditionally trigger `select-folder` trigger.

Reply:
got it, yeah conditionally emitting sounds better